contribs/src/main/java/com/netflix/conductor/contribs/http/GenericHttpTask.java

```diff
@@ -130,8 +132,16 @@ HttpResponse httpCall(Input input, Task task, Workflow workflow, WorkflowExecuto
 
 		// Attach the Authorization header by adding entry to the input's headers
 		if (input.isAuthorize()) {
-			setAuthorization(input);
-			setAuthorizationContext(input, workflow.getAuthorizationContext());
+			setAuthToken(input);
+		}
+
+		// Attach Authorization-Context: {SSO token} if present and enabled
+		if (authContextEnabled) {
+			if(StringUtils.isNotEmpty(workflow.getContextToken())) {
```

Comment:
If using `StringUtils` here, this could be simplified to:
```
input.getHeaders().put(CommonParams.AUTH_CONTEXT, StringUtils.defaultIfEmpty(workflow.getContextToken(), ""));
```

Reply:
Right!